BUILD.md

```diff
@@ -16,8 +16,8 @@ The following packages must be installed before Avro can be built:
  - Perl: Perl 5.24.1 or greater, gmake, Module::Install,
    Module::Install::ReadmeFromPod, Module::Install::Repository,
    Math::BigInt, JSON::XS, Try::Tiny, Regexp::Common, Encode,
-   IO::String, Object::Tiny, Compress::ZLib, Test::More,
-   Test::Exception, Test::Pod
+   IO::String, Object::Tiny, Compress::ZLib, Error::Simple,
```

Comment:
I don't see `Error:Simple` in the Dockerfile: https://github.com/apache/avro/blob/master/share/docker/Dockerfile#L82-L87

Reply:
That's OK, because it's already installed in the openjdk:8 image :)

```
$ docker run -it openjdk:8 perl -MError::Simple -e 1
$ # "Can't locate" error doesn't occur
```

Reply:
Okay, cool. I'm not too familiar with Perl :-)